### DISCUSSION TITLE
How to write tpv29-receivers.dat

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#608] [Q&A] [answered]
May I ask how these two documents `tpv29-faultreceivers.dat` and 
`tpv29-receivers.dat`were written, and are there any reference books or websites? I see a bunch of data, I don't know the basis for this.
![image](https://user-images.githubusercontent.com/104721358/178277378-c8f4766f-7b27-43a8-91e4-1c60bf72123f.png)

**Comment by krenzland** [comment, marked as answer]
Can you please in the future first try and find an answer in our documentation? This question for example is clearly answered in https://seissol.readthedocs.io/en/latest/fault-output.html

**Reply by Thomas-Ulrich** [reply]
Hi, have a look also here: https://strike.scec.org/cvws/tpv29_30docs.html
That's the description of the SCEC benchmark.
The location of the receiver should be described there.

**Reply by daisy20170101** [reply]
the three columns are x, y, z in meter, respectively.

**Reply by smallriver666** [reply]
> Can you please in the future first try and find an answer in our documentation? This question for example is clearly answered in https://seissol.readthedocs.io/en/latest/fault-output.html

I'm very sorry to bother you, because I haven't used GitHub before, so I'm not very familiar with it, and I'm just making up my theory, so I may ask some stupid questions, I'm sorry. But in the end I appreciate your patience in replying to me.
